deploy/crd/aquasecurity.github.io_clustercompliancereports.yaml

```diff
@@ -0,0 +1,153 @@
+          resource.
+        properties:
+          apiVersion:
+            description: 'APIVersion defines the versioned schema of this representation
+              of an object. Servers should convert recognized schemas to the latest
+              internal value, and may reject unrecognized values. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#resources'
+            type: string
+          kind:
+            description: 'Kind is a string value representing the REST resource this
+              object represents. Servers may infer this from the endpoint the client
+              submits requests to. Cannot be updated. In CamelCase. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#types-kinds'
+            type: string
+          metadata:
+            type: object
+          spec:
+            description: ReportSpec represent the compliance specification
+            properties:
+              controls:
+                items:
+                  description: Control represent the cps controls data and mapping
+                    checks
+                  properties:
+                    defaultStatus:
+                      type: string
+                    description:
+                      type: string
+                    id:
+                      type: string
+                    kinds:
+                      items:
+                        type: string
+                      type: array
+                    mapping:
+                      description: Mapping represent the scanner who perform the control
+                        check
+                      properties:
+                        checks:
+                          items:
+                            description: SpecCheck represent the scanner who perform
+                              the control check
+                            properties:
+                              id:
+                                type: string
+                            required:
+                            - id
+                            type: object
+                          type: array
+                        scanner:
+                          type: string
+                      required:
+                      - checks
+                      - scanner
+                      type: object
+                    name:
+                      type: string
+                    severity:
+                      description: Severity level of a vulnerability or a configuration
+                        audit check.
+                      type: string
+                  required:
+                  - id
+                  - kinds
+                  - mapping
+                  - name
+                  - severity
+                  type: object
+                type: array
+              cron:
+                type: string
+              description:
+                type: string
```

Comment:
same here : 
  cron:
                  type: string
                  pattern: '^(((([\*]{1}){1})|((\*\/){0,1}(([0-9]{1}){1}|(([1-5]{1}){1}([0-9]{1}){1}){1}))) ((([\*]{1}){1})|((\*\/){0,1}(([0-9]{1}){1}|(([1]{1}){1}([0-9]{1}){1}){1}|([2]{1}){1}([0-3]{1}){1}))) ((([\*]{1}){1})|((\*\/){0,1}(([1-9]{1}){1}|(([1-2]{1}){1}([0-9]{1}){1}){1}|([3]{1}){1}([0-1]{1}){1}))) ((([\*]{1}){1})|((\*\/){0,1}(([1-9]{1}){1}|(([1-2]{1}){1}([0-9]{1}){1}){1}|([3]{1}){1}([0-1]{1}){1}))|(jan|feb|mar|apr|may|jun|jul|aug|sep|okt|nov|dec)) ((([\*]{1}){1})|((\*\/){0,1}(([0-7]{1}){1}))|(sun|mon|tue|wed|thu|fri|sat)))$'
                  description: 'cron define the intervals for report generation'